cfme/tests/cloud/test_tenant_quota.py

```diff
@@ -49,18 +49,12 @@ def template_name(provisioning):
     return provisioning["image"]["name"]
 
 
-@pytest.fixture(scope="module")
-def roottenant(appliance):
-    # configuration tenants, not cloud tenants
-    return appliance.collections.tenants.get_root_tenant()
-
-
 @pytest.fixture
-def prov_data(vm_name, template_name):
+def prov_data(vm_name, template_name, provisioning):
     return {
         "catalog": {'vm_name': vm_name, 'catalog_name': {'name': template_name}},
         "environment": {'automatic_placement': True},
-        'properties': {'instance_type': partial_match('m1.large')}
+        'properties': {'instance_type': partial_match(provisioning["instance_type2"])}
```

Comment:
Thoughts on making a `.get()` call here? If another OSP provider is added without this key, it is going to fail on a `KeyError`